pkg/cli/cli.go

```diff
@@ -144,7 +144,7 @@ func buildServiceBuilder(builder client.ServiceBuilder) serviceBuilder {
 			ErrOut:         c.ErrOrStderr(),
 			RetryAttempts:  3,
 			RetryDelay:     5 * time.Second,
-			ChunkSizeBytes: 10 * 1024 * 1024,
+			ChunkSizeBytes: 16 * 1024 * 1024,
```

Comment:
Please extract this change in constant value, along with the one in client.go to a separate commit and use the "rebase and merge" strategy instead of "squash and merge"

Reply:
done, in client.go we already use a constant variable.